### PR TITLE
Port Guestbook tests to mutiarch

### LINKED ISSUE #52232: Port Guestbook tests to mutiarch
**Is this a BUG REPORT or FEATURE REQUEST?**:

/kind feature

**What happened**:
Part of https://github.com/kubernetes/examples/pull/43 guestbook application has been ported for multiarch, so this workitem to use those images and make tests multi architectures to run on different architectures like amd64, arm, arm64, ppc64le etc..
**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration**:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by mkumatag**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #52232 

**Special notes for your reviewer**:

**Release note**:

```NONE
NONE
```

### CODE REVIEW

File: test/utils/image/manifest.go [hunk 1]
```diff
@@ -25,6 +25,7 @@ const (
 	e2eRegistry     = "gcr.io/kubernetes-e2e-test-images"
 	gcRegistry      = "gcr.io/google-containers"
 	PrivateRegistry = "gcr.io/k8s-authenticated-test"
+	sampleRigistry  = "gcr.io/google-samples"
```
Comment (luxas) [inline test/utils/image/manifest.go]: sampleRegistry

File: test/e2e/testing-manifests/guestbook/redis-slave-service.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    app: redis
+    role: slave
+    tier: backend
+spec:
+  ports:
+  - port: 6379
+  selector:
+    app: redis
+    role: slave
+    tier: backend
```
Comment (luxas) [inline test/e2e/testing-manifests/guestbook/redis-slave-service.yaml]: nit: add newline

**Review by luxas — COMMENTED** [review]
Thanks @mkumatag!
Some small nits (like adding newlines at the end of the yaml files), otherwise LGTM

I consider this a bug fix to get this guestbook e2e green for our other architectures. Also, this only touches the e2e binary.

@ixdy Are you ok with approving this bug fix?

**Comment by mkumatag** [comment]
@luxas fixed all the nits, can you PTAL at latest commit.

**Review by luxas — APPROVED** [review]
/lgtm

**Comment by mkumatag** [comment]
@ixdy we need this PR to make 100% conformance test pass for multi-arch, please approve.

File: test/utils/image/manifest.go [hunk 3]
```diff
@@ -25,6 +25,7 @@ const (
 	e2eRegistry     = "gcr.io/kubernetes-e2e-test-images"
 	gcRegistry      = "gcr.io/google-containers"
 	PrivateRegistry = "gcr.io/k8s-authenticated-test"
+	sampleRegistry  = "gcr.io/google-samples"
```
Comment (ixdy) [inline test/utils/image/manifest.go]: nit: `samplesRegistry`

**Pull request merged**